scikeras/utils/transformers.py

```diff
@@ -0,0 +1,194 @@
+            target_type == "multilabel-indicator"
+            and y.min() == 0
+            and (y.sum(axis=1) == 1).all()
+        ):
+            target_type = "multiclass-onehot"
+
+        self.n_outputs_ = 1
+        self.model_n_outputs_ = 1
+        self.y_dtype_ = y.dtype
+        self._target_type = target_type
+
+        if target_type in ("binary", "multiclass"):
+            self.classes_ = self.final_encoder_[1].categories_[0]
+            self.n_classes_ = self.classes_.size
+        elif target_type in ("multiclass-onehot", "multilabel-indicator"):
+            self.classes_ = np.arange(0, y.shape[1])
+            self.n_classes_ = y.shape[1]
+        elif target_type == "multiclass-multioutput":
+            self.classes_ = [np.unique(y[:, j]) for j in range(y.shape[1])]
+            self.n_classes_ = len(self.n_classes_)
+
+        return self
+
+    def transform(self, X: np.ndarray) -> np.ndarray:
+        y = X  # rename for clarity, the input is always expected to be a target `y`
+        # no need to validate n_outputs_ or model_n_outputs_, those are hardcoded
+        # self.classes_ and self.n_classes_ are validated by the transformers themselves
+        return self.final_encoder_.transform(y)
+
+    def inverse_transform(
+        self, X: np.ndarray, return_proba: bool = False
+    ) -> np.ndarray:
+        y = X  # rename for clarity, the input is always expected to be a target `y`
+        if self._target_type == "binary":
+            # array([0.9, 0.1], [.2, .8]) -> array(['yes', 'no'])
+            if y.ndim == 1 or (y.shape[1] == 1 and self.n_classes_ == 2):
+                # result from a single sigmoid output
+                # reformat so that we have 2 columns
+                y = np.column_stack([1 - y, y])
+            y_ = np.argmax(y, axis=1).reshape(-1, 1)
+            class_predictions = self.final_encoder_.inverse_transform(y_)
+        elif self._target_type == "multiclass":
+            # array([0.8, 0.1, 0.1], [.1, .8, .1]) ->
+            # array(['apple', 'orange'])
+            idx = np.argmax(y, axis=-1)
+            if not is_categorical_crossentropy(self.loss):
+                y_ = idx.reshape(-1, 1)
+            else:
+                y_ = np.zeros(y.shape, dtype=int)
+                y_[np.arange(y.shape[0]), idx] = 1
+            class_predictions = self.final_encoder_.inverse_transform(y_)
+        elif self._target_type == "multiclass-onehot":
+            # array([.8, .1, .1], [.1, .8, .1]) ->
+            # array([[1, 0, 0], [0, 1, 0]])
+            idx = np.argmax(y, axis=-1)
+            y_ = np.zeros(y.shape, dtype=int)
+            y_[np.arange(y.shape[0]), idx] = 1
+            class_predictions = y_
+        elif self._target_type == "multilabel-indicator":
+            class_predictions = np.around(y)
+        else:
+            raise NotImplementedError(
+                "Class-predictions are not clearly defined for"
+                " 'multiclass-multioutput' target types."
```

Comment:
What should the user do to resolve this issue?

Reply:
Thanks for picking it up. I typed something in and planned to exapand on it but forgot. I plan on:
```python3
raise NotImplementedError(
    f"Class-predictions are not clearly defined for."
     " 'multiclass-multioutput' target types."
    "\n\nTo implement support, subclass KerasClassifier and override"
    " `target_transformer` with a transformer that supports this"
    " label type."
)
```

Reply:
Implemented longer message. Resolving.